doc/operators_precedence.rst

```diff
@@ -0,0 +1,56 @@
+Unary operators precedence:
+
+========== ========
+Precedence Operator
+========== ========
+
+500         -
+            +
+70          not
+0           (
+
+Binary and Ternary operators precedence:
+
+========== ========
+Precedence Operator
+========== ========
+
+300         |
+            .
+            [
+            (
```

Comment:
I think this needs more explanation to indicate that this parenthesis is representing the function call. I know no language that describe function calls as being a binary operator

Reply:
this precedence looks weird to me compared to unary operators. How is `- myfunc()` parsed, before and after this change ? Same for `- var[0]`

Reply:
Btw, what happens if an extension defines an operator with higher precedence that this "function call" operator ? Using a function call near this operator would suddenly not parse the function call in the expected way.

Reply:
Next step is to add a specific test suite for operators.

Reply:
I've just pushed more tests and changed the precedence of the filter operator to make an expression like ``{{ -1|abs }}`` works as expected.